packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -223,13 +228,19 @@ class DateTimePicker extends React.Component {
         updatedDateTime = DateTimeUtils.updateTime(this.state.dateTime, this.timeValue, this.props.showSeconds);
       }
 
-      this.checkAmbiguousTime(updatedDateTime);
-      this.handleBlur(event, isDateTimeValid);
+      this.checkAmbiguousTime(updatedDateTime, () => {
+        // If the entered time is ambiguous then do not handle blur just yet. It should be handled _after_
+        // the ambiguity is resolved (i.e., after dismissing the Time Clarification dialog).
+        if (!(this.state.isAmbiguousTime && this.state.isTimeClarificationOpen)) {
+          this.handleBlur(event, updatedDateTime);
+        }
+      });
     }
   }
 
-  handleBlur(event, isCompleteDateTime) {
+  handleBlur(event, momentDateTime) {
     if (this.props.onBlur) {
+      const isCompleteDateTime = DateTimeUtils.isValidDateTime(this.dateValue, this.timeValue, this.state.dateFormat);
```

Comment:
The `showSeconds` prop is need to be passed as the 4th parameter to `DateTimeUtils.isValidDateTime` to correctly validate the time value to see if the seconds precision is included.

Reply:
Ah, thanks for the heads-up. It looks like I missed the recent change that went into master. Fixed it here: https://github.com/cerner/terra-framework/pull/784/commits/1eaae53d54c7058dd14309b33fd5e0879cea9dd4